build.gradle

```diff
@@ -71,3 +72,7 @@ subprojects {
 
     apply from: '../gradle/publish.gradle'
 }
+
+configure(subprojects.findAll {it.name != 'jaeger-generated'}) {
+   apply plugin: 'net.ltgt.errorprone'
```

Comment:
Error prone has an [undocumented](https://github.com/google/error-prone/issues/463) flag to ignore generated code that has an `@Generated` annotation. 
Thrift 0.9.3 has a bug where the `@Generated` annotation isn't applied to classes that only contain enums. (e.g.  `com.twitter.zipkin.thriftjava.AnnotationType`)